mdata/aggregator.go

```diff
@@ -9,16 +9,21 @@ type AggSetting struct {
 	Span      uint32 // in seconds, controls how many input points go into an aggregated point.
 	ChunkSpan uint32 // duration of chunk of aggregated metric for storage, controls how many aggregated points go into 1 chunk
 	NumChunks uint32 // number of chunks to keep in memory. remember, for a query from now until 3 months ago, we will end up querying the memory server as well.
-	ttl       uint32 // how many seconds to keep the chunk in cassandra
+	TTL       uint32 // how many seconds to keep the chunk in cassandra
 	Ready     bool   // ready for reads?
 }
 
+type AggSettings struct {
+	TTL  uint32       // default TTL for raw data
```

Comment:
this field should be `RawTTL` or something because it easily gets confusing otherwise. btw why does it say "default TTL"? this value is whatever  the user asked for, not any default.